<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BABA IS YOU</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>381</width>
      <height>381</height>
     </rect>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>186</red>
       <green>189</green>
       <blue>182</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>136</red>
       <green>138</green>
       <blue>133</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadLevelButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload Level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAndLeaveButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and leave</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageBoxLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>611</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadOrSaveButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yesButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>600</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
